packages/form/components/controls/RadioGroup.astro

```diff
@@ -6,9 +6,10 @@ import type { Radio, ControlOption } from '@astro-reactive/common';
 
 export interface Props {
 	control: Radio;
+	readOnly?: Boolean;
```

Comment:
This should be `boolean`. `Boolean` type isn't not assignable in `readOnly` and `disabled` input attribute.

```suggestion
	readOnly?: boolean;
```

Reply:
Thanks @Icyscools for the review, and great catch too. `Boolean` with a capital B is actually an object :)

Reply:
Done :D